clang/lib/StaticAnalyzer/Checkers/DynamicTypePropagation.cpp

```diff
@@ -609,9 +617,13 @@ storeWhenMoreInformative(ProgramStateRef &State, SymbolRef Sym,
 /// symbol and the destination type of the cast are unrelated, report an error.
 void DynamicTypePropagation::checkPostStmt(const CastExpr *CE,
                                            CheckerContext &C) const {
+  ProgramStateRef State = C.getState();
+  ExplodedNode *AfterTypeProp = dynamicTypePropagationOnCasts(CE, State, C);
+
   if (CE->getCastKind() != CK_BitCast)
     return;
 
+  ASTContext &ASTCtxt = C.getASTContext();
```

Comment:
Do we need to move this?

Reply:
I guess, it's a matter of taste, but I can see that keeping it closer to the use point makes sense.
Moved it back where it was.